aunt-leahs-app/src/pages/VolunteerLoginPage.jsx

```diff
@@ -4,23 +4,15 @@ import { connect } from 'react-redux';
 import { compose } from 'redux';
 
 import { pages } from '../constants';
-
-import SelectBox from '../components/SelectBox'
-import CustomButton from '../components/CustomButton'
-import Header from '../components/Header'
+import CustomButton from '../components/customButton'
+import Head from '../components/header'
```

Comment:
Does this actually run? You imported `Head` but used `Header` below on line 36

Reply:
This could be due to incorrectly resolving merging conflicts. For this to compile, the lines should be

`import CustomButton from '../components/CustomButton'`

`import Header from '../components/Header'`